app/src/org/commcare/dalvik/activities/DispatchActivity.java

```diff
@@ -0,0 +1,311 @@
+package org.commcare.dalvik.activities;
```

Comment:
Corrupt dialogs can be shown on using this activity. I tested it and it displayed fine, see http://manage.dimagi.com/default.asp?189955 for an screenshot.
I wanted to avoid adding any inflation logic, because it really shouldn't _usually_ show anything.